platform/swift/source/Logger.swift

```diff
@@ -128,6 +128,7 @@ public final class Logger {
         let appStateAttributes = AppStateAttributes()
         let clientAttributes = ClientAttributes()
         let deviceAttributes = DeviceAttributes()
+        let hardwareVersion = deviceAttributes.hardwareVersion
```

Comment:
is this allocation necessary? seems like we can call `model: deviceAttributes.hardwareVersion` below

Reply:
I'm not very familiar with how swift works and what allocates or not. I will read more and clean it up.